render/filters.go

```diff
@@ -293,7 +293,14 @@ func IsNotPseudo(n report.Node) bool {
 // IsNamespace checks if the node is a pod/service in the specified namespace
 func IsNamespace(namespace string) FilterFunc {
 	return func(n report.Node) bool {
-		gotNamespace, _ := n.Latest.Lookup(kubernetes.Namespace)
+		try_keys := []string{kubernetes.Namespace, docker.LabelPrefix + "io.kubernetes.pod.namespace"}
```

Comment:
Is `"io.kubernetes.pod.namespace"` is used at render/filters.go too, please use a common constant and/or a function to extract it from a node.